pkg/ovsdb/ovsdb.go

```diff
@@ -97,13 +97,13 @@ func (self *OvsDriver) ovsdbTransact(ops []libovsdb.Operation) ([]libovsdb.Opera
 
 // **************** OVS driver API ********************
 // Create an internal port in OVS
-func (self *OvsBridgeDriver) CreatePort(intfName, contNetnsPath, contIfaceName, ovnPortName string, vlanTag uint) error {
+func (self *OvsBridgeDriver) CreatePort(intfName, contNetnsPath, contIfaceName, ovnPortName string, vlanTags []uint, portType string) error {
```

Comment:
I would keep the single vlan tag as well as adding the array. So we can support in the future 'native tagged' and 'native untagged' vlan modes. Which are combination of trunk and access modes.

http://www.openvswitch.org/support/dist-docs/ovs-vswitchd.conf.db.5.txt - openvswitch documentation explaining the vlan modes (in VLAN Configuration section).

Reply:
Yes, changed the method to keep both vlan tag and trunks parameters.